portfolioScript.js

```diff
@@ -17,3 +17,157 @@ document.getElementById('clsebtn').addEventListener('click', () => {
+    imgsrc: 'images/Snapshoot.png',
+    imgMain: 'Backgroud-images/Img_Plaholder.png',
+    technologies: ['css', 'Javascript', 'html'],
+    techMain: ['css', 'Javascript', 'html'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  },
+];
+
+const container = document.getElementById('whole-section');
+
+const works = document.createElement('div');
+works.setAttribute('class', 'desk-works');
+
+for (let i = 0; i < object.length; i += 1) {
+  const read = `
+  <section class="works-container">
+  <img class="image-grey" src="${object[i].imgMain}" alt="image-of-works">
+  <div class="multi">
+      <section class="multi-post">
+          <h2 class="multi-post-title">${object[i].title}</h2>
+          <section class="buttons">
+          ${object[i].techMain.map((language) => `<button class="languages">${language}</button>`).join('')}
+          </section>
+          <button class="see-project" id="${i}">See Project ${i + 1}</button>
+       </section>
+   </div>
+  </section>`;
+
+  works.innerHTML += read;
+}
+
+container.appendChild(works);
+
+// render for pop up
+
+// Get the button that opens the modal
+const projectBtns = document.querySelectorAll('.see-project');
+const body = document.getElementById('body');
+
+const overlay = document.createElement('div');
+overlay.setAttribute('class', 'overlay');
+overlay.setAttribute('id', 'overlay');
+
+const popup = document.createElement('div');
+popup.setAttribute('class', 'pop-up-modal');
+overlay.appendChild(popup);
+
+projectBtns.forEach((button) => {
+  button.addEventListener('click', () => {
+    const btnId = button.id;
+    const content = `
+        <div class="close-btn-cont">
+            <button class="cerrar" id="cerrar">&times</button>
+        </div>
+        <div class="img-cont-pop">
+            <img src="${object[btnId].imgsrc}" alt="..." id="img">
+        </div>
+        
+        <div class="space-bet">
+            <h2 id="name">${object[btnId].name}</h2>
+            <div class="flex-btn">
+                <div>
+                    <button class="live" id="demo">See Live <img src="images/circle.png" alt="live"></button>
+                </div>
+                <div>
+                    <button class="live" id="source">Source <img src="images/cat.png" alt="liveCat"></button>
+                </div>
+            </div>
+
+        </div>
+        <section class="buttons-tech" id="buttons">
+          ${object[btnId].technologies.map((language) => `<button class="languages">${language}</button>`).join('')}
+        </section>
```

Comment:
- [OPTIONAL] Kindly change the list of languages that are used on this project to a ul because they are not buttons but only a list of languages that were used on this project. I suggest that you don't use a section on this part because you can say that section is all modal and for only this list you can use a div. Keep coding 💪

Reply:
thank you, its done :smile: